import-scripts/fetch-dmp-data-for-import.sh

```diff
@@ -188,6 +185,10 @@ MY_FLOCK_FILEPATH="/data/portal-cron/cron-lock/fetch-dmp-data-for-import.lock"
         # fetch new/updated IMPACT samples using CVR Web service   (must come after git fetching)
         drop_dead_instant_step=$(date --date="+3hours" -Iseconds) # nearly 3 hours from now
         drop_dead_instant_string=$(find_earlier_instant "$drop_dead_instant_step" "$DROP_DEAD_INSTANT_END_TO_END")
+        
```

Comment:
I think the comment before the function call is enough of a separator. I think I'd prefer to remove the blank line before and after the call to the preconsume script.